packages/xrpl/HISTORY.md

```diff
@@ -3,6 +3,7 @@
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 
 ## Unreleased
+* Add AMM integration tests
```

Comment:
This doesn't need to be in the history since it doesn't affect how people use it.